feature/experimental/policy/policy_vrf_selection/otg_tests/protocol_dscp_rules_for_vrf_selection_test/protocol_dscp_rules_for_vrf_selection_test.go

```diff
@@ -231,6 +237,9 @@ func getSubInterface(dutPort *attrs.Attributes, index uint32, vlanID uint16) *oc
 
 // configInterfaceDUT configures the interface with the Addrs.
 func configInterfaceDUT(i *oc.Interface, dutPort *attrs.Attributes) *oc.Interface {
+	if *deviations.InterfaceEnabled {
```

Comment:
Do you still need `!*deviations.IPv4MissingEnabled` here?

Reply:
Yes, Here it is required at interface level, like it is done in ATE script. It seems it was missed for this OTG script earlier. Other places, deviation is used to enable subinterface and protocol level, here it is used enable at interface-level.